src/qml/controls/OptionSwitch.qml

```diff
@@ -7,21 +7,23 @@ import QtQuick.Controls 2.15
 
 Switch {
     id: root
-    indicator: Rectangle {
-        implicitWidth: 45
-        implicitHeight: 28
-        x: root.leftPadding
-        y: Math.round((parent.height - height) / 2)
-        radius: 18
+    implicitWidth: 45
+    implicitHeight: 28
+    background: Rectangle {
+        radius: Math.floor(height / 2)
         color: root.checked ? Theme.color.orange : Theme.color.neutral4
-        Rectangle {
-            id: indicatorButton
-            y: parent.height / 2 - height / 2
-            x: root.checked ? parent.width - width - 4 : 0 + 4
-            width: 20
-            height: 20
-            radius: 18
-            color: Theme.color.white
+    }
+    indicator: Rectangle {
+        property real _margin: Math.round((parent.height - height) / 2)
+        y: Math.round(parent.height - height) / 2
+        x: root.checked ? parent.width - width - _margin : _margin
+        radius: 10
```

Comment:
```suggestion
        radius: 18
```

Though changing this value does not indicate visual difference, I think it is still better to make this change as this matches the value given in the [design Figma file](https://www.figma.com/file/GaCoOSNHB2yMB9ThiDtred/Bitcoin-Core-App).

Reply:
> 10 makes no sense though?

Reply:
Yes, it doesn't, and I agree with that.
But I feel a little iffy about having the value inconsistent with the design file.

However, this is just a tiny nit and can be safely ignored if you want.

Reply:
@GBKS thoughts?

Reply:
In Figma, the small circle in the toggle is 20px tall, the big one is 28px tall. So the radii of 10px and 14px would be the minimum.

I sometimes set them to 500px or some other ridiculously high number because it doesn't matter in Figma if you're over the exact value. But if you want to be precise, then half the height is what it should be. Looks like for the smaller size, I simply kept the value of the larger one. 10px would be the precise number.